### PR TITLE
Lbryinc errors

### PR DESCRIPTION
**Pull request by andybeletsky**
Need to be able to tell connectivity errors from API errors — right now they're all lumped together, which doesn't look good on the SLI graph.

### CODE REVIEW

File: schema/address/validate.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ const lbrycrdTestnet = "lbrycrd_testnet"
 const lbrycrdRegtest = "lbrycrd_regtest"
 
 var addressPrefixes = map[string][2]byte{
-	lbrycrdMain:    [2]byte{lbrycrdMainPubkeyPrefix, lbrycrdMainScriptPrefix},
-	lbrycrdTestnet: [2]byte{lbrycrdTestnetPubkeyPrefix, lbrycrdTestnetScriptPrefix},
-	lbrycrdRegtest: [2]byte{lbrycrdRegtestPubkeyPrefix, lbrycrdRegtestScriptPrefix},
+	lbrycrdMain:    {lbrycrdMainPubkeyPrefix, lbrycrdMainScriptPrefix},
```
Comment (nikooo777) [inline schema/address/validate.go:27]: ouch did you test this out? I'm always very scared of touching those things

**Comment by nikooo777** [comment]
@lyoshenka can you check this out too please? Maybe you'll spot something I don't see

File: schema/address/validate.go (hunk 1 above)
Comment (andybeletsky) [reply schema/address/validate.go:27]: This syntax is fine (and it's also an auto gofmt fix)

Comment (lyoshenka) [reply schema/address/validate.go:27]: [fear is the mind-killer](https://dune.fandom.com/wiki/Litany_Against_Fear)

File: extras/lbryinc/client.go [hunk 2]
```diff
@@ -46,6 +46,15 @@ type APIResponse struct {
 	Data    *ResponseData `json:"data"`
 }
 
+// APIError wraps errors returned by LBRY API server to discern them from other kinds (like http errors).
+type APIError struct {
+	Err error
```
Comment (lyoshenka) [inline extras/lbryinc/client.go:51]: `Err` can be `err`

**Review by lyoshenka — APPROVED** [review]

**Pull request merged**